.env.example

```diff
@@ -80,4 +80,11 @@ DACO_UI_APPLICATION_SECTION_PATH=/applications/{id}?section={section}
 ##############
 # Misc       #
 ##############
-FILE_UPLOAD_LIMIT=#in bytes x * 1024 * 1024
\ No newline at end of file
+FILE_UPLOAD_LIMIT=#in bytes x * 1024 * 1024
+
+##############
+# DURATIONS  #
+##############
+DAYS_TO_EXPIRY_1=
```

Comment:
are these used anywhere now ?

Reply:
Yeah they're still used for calculating the timing of the first 2 emails (once that part gets hooked up)